### PR TITLE
Add NVD scanning to SQL LRS workflow

### PR DESCRIPTION
**Pull request by kelvinqian00**

### CODE REVIEW

**Comment by milt** [comment]
Don't forget a scheduled one too when you come back to this.

**Comment by kelvinqian00** [comment]
> Don't forget a scheduled one too when you come back to this.

That was what I did in https://github.com/yetanalytics/lrsql/pull/197/commits/2a86d797a94c724eb9b7a8f78f40da11c0e85786.

**Comment by kelvinqian00** [comment]
^ Going to keep the local alias + Make target around in order to make local dev easy.

File: .github/workflows/nvd.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: NVD Scan
```
Comment (milt) [inline .github/workflows/nvd.yml:1]: You should specify here (if not in the filename) that this is a schedule scan

File: deps.edn [hunk 2]
```diff
@@ -133,7 +133,7 @@
   {:extra-deps  {com.github.seancorfield/depstar {:mvn/version "2.1.267"}}
    :extra-paths ["src/build"]} 
   :nvd
-  {:replace-deps {nvd-clojure/nvd-clojure {:mvn/version "1.9.0"}}
+  {:replace-deps {nvd-clojure/nvd-clojure {:mvn/version "2.0.0"}}
```
Comment (milt) [inline deps.edn:136]: I believe the scanner provides nvd itself, so no need to have this here (unless there is some other call in the makefile that needs it)

**Review by milt — APPROVED** [review]
Looks good, see minor notes

File: deps.edn (hunk 2 above)
Comment (kelvinqian00) [reply deps.edn:136]: It's for the `check-vuln` Makefile target. Neither this nor `check-vuln` are strictly necessary anymore, but as I said above I want to keep it for easy manual dev. Another way would be to get rid of this alias but keep `check-vuln`, having the latter use clojure.tools, but that would require the _user_ to ensure that they have nvm-clojure installed.

File: .github/workflows/nvd.yml (hunk 1 above)
Comment (kelvinqian00) [reply .github/workflows/nvd.yml]: https://github.com/yetanalytics/lrsql/pull/197/commits/fea48c7d6e6af3eacdee27cad067f6ca83fc8d90

**Pull request merged**